developer/src/inst/node/kmc.cmd

```diff
@@ -2,11 +2,24 @@
 rem This script avoids path dependencies for node for distribution
 rem with Keyman Developer. When used on platforms other than Windows,
 rem node can be used directly with the compiler (`npm link` will setup).
+
+set root=%~dp0
+
 if "%1" == "--enable-source-maps" (
   set ESM=--enable-source-maps
   shift
 ) else (
   set ESM=
 )
-"%~dp0\node.js\node.exe" %ESM% "%~dp0\kmc\kmc.mjs" %1 %2 %3 %4 %5 %6 %7 %8 %9
```

Comment:
Amusingly, this would not have worked in the past with `--enable-source-maps` parameter anyway, because `shift` would have meant `%~dp0` was actually the path portion of `"--enable-source-maps"`...